tools/reload_sdk_allowlist.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+SHELL_FOLDER="$(cd "$(dirname $0)" || exit;pwd)"
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5822413)